bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/api/dto/Loadpoint.java

```diff
@@ -0,0 +1,316 @@
+    private int targetSoC;
+
+    @SerializedName("targetTime")
+    private String targetTime;
+
+    @SerializedName("title")
+    private String title;
+
+    @SerializedName("vehicleCapacity")
+    private long vehicleCapacity;
+
+    @SerializedName("vehicleOdometer")
+    private double vehicleOdometer;
+
+    @SerializedName("vehiclePresent")
+    private boolean vehiclePresent;
+
+    @SerializedName("vehicleRange")
+    private long vehicleRange;
+
+    @SerializedName("vehicleSoC")
+    private int vehicleSoC;
+
+    @SerializedName("vehicleTitle")
+    private String vehicleTitle;
+
+    /**
+     * @return the active phases
+     */
+    public int getActivePhases() {
+        return activePhases;
+    }
+
+    /**
+     * @return the charge current
+     */
+    public double getChargeCurrent() {
+        return chargeCurrent;
+    }
+
+    /**
+     * @return the charge duration
+     */
+    public long getChargeDuration() {
+        return chargeDuration;
+    }
+
+    /**
+     * @return the charge power
+     */
+    public double getChargePower() {
+        return chargePower;
+    }
+
+    /**
+     * @return the charge remaining duration
+     */
+    public long getChargeRemainingDuration() {
+        return chargeRemainingDuration;
+    }
+
+    /**
+     * @return the chargeRemainingEnergy
+     */
+    public double getChargeRemainingEnergy() {
+        return chargeRemainingEnergy;
+    }
+
+    /**
+     * @return the charged energy
+     */
+    public double getChargedEnergy() {
+        return chargedEnergy;
+    }
+
+    /**
+     * @return wtether loadpoint is charging a vehicle
+     */
+    public boolean getCharging() {
+        return charging;
+    }
+
+    /**
+     * @return whether a vehicle is connected to the loadpoint
+     */
+    public boolean getConnected() {
+        return connected;
+    }
+
+    /**
+     * @return the connected duration
+     */
+    public long getConnectedDuration() {
+        return connectedDuration;
+    }
+
+    /**
+     * @return whether loadpoint is enabled
+     */
+    public boolean getEnabled() {
+        return enabled;
+    }
+
+    /**
+     * @return whether loadpoint has vehicle
+     */
+    public boolean getHasVehicle() {
+        return hasVehicle;
+    }
+
+    /**
+     * @return the loadpoint
+     */
+    public int getLoadpoint() {
+        return loadpoint;
+    }
+
+    /**
+     * @return the maximum current
+     */
+    public double getMaxCurrent() {
+        return maxCurrent;
+    }
+
+    /**
+     * @return the minimum current
+     */
+    public double getMinCurrent() {
+        return minCurrent;
+    }
+
+    /**
+     * @return the minimum state of charge
+     */
+    public int getMinSoC() {
+        return minSoC;
+    }
+
+    /**
+     * @return the mode: off, now, minpv, pv
+     */
+    public String getMode() {
+        return mode;
+    }
+
+    /**
+     * @return the phases
+     */
+    public int getPhases() {
```

Comment:
Is it number of phases being returned?

Reply:
Okay - consider if this is worth mentioning in the method Javadoc and/or method name.